.npmrc

```diff
@@ -1,11 +1,10 @@
 engine-strict=true
 frozen-lockfile=true
 strict-peer-dependencies=true
+link-workspace-packages=true
```

Comment:
changes to pnpm defaults require this for workspace deps to function during install now.